Failed tests (10340): TestBuildInfo

The following test appears to have failed:

[#10340](https://circleci.com/gh/cockroachdb/cockroach/10340):

```
  pprof: docker exec -it 100fc /bin/bash -c 'go tool pprof /cockroach <(wget --no-check-certificate -qO- https://$(hostname):26257/debug/pprof/heap)'
E1224 01:41:17.096466 18782 acceptance/cluster/localcluster.go:395  node=0 status=create
E1224 01:41:17.096498 18782 acceptance/cluster/localcluster.go:395  node=0 status=start
I1224 01:41:18.107224 18782 acceptance/main_test.go:166  invalid character 'p' after top-level value
I1224 01:41:18.207642 18782 acceptance/cluster/localcluster.go:522  stopping
--- FAIL: TestBuildInfo (11.48s)
panic: interface conversion: interface {} is nil, not map[string]interface {} [recovered]
	panic: interface conversion: interface {} is nil, not map[string]interface {}

goroutine 21 [running]:
testing.tRunner.func1(0xc8201761b0)
	/usr/local/go/src/testing/testing.go:450 +0x171
github.com/cockroachdb/cockroach/acceptance.checkGossip.func1(0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/acceptance/gossip_peerings_test.go:57 +0x4ec
github.com/cockroachdb/cockroach/util.SucceedsWithinDepth(0x1, 0x7f8b48064878, 0xc8201761b0, 0x4a817c800, 0xc8201ebe48)
	/go/src/github.com/cockroachdb/cockroach/util/testing.go:147 +0x18f
github.com/cockroachdb/cockroach/util.SucceedsWithin(0x7f8b48064878, 0xc8201761b0, 0x4a817c800, 0xc8201ebe48)

```
Please assign, take a look and update the issue accordingly.

--- a/acceptance/gossip_peerings_test.go
+++ b/acceptance/gossip_peerings_test.go
@@ -19,6 +19,7 @@
 package acceptance
 
 import (
+	"errors"
 	"math/rand"
 	"strings"
 	"testing"
@@ -54,7 +55,10 @@ func checkGossip(t *testing.T, c cluster.Cluster, d time.Duration,
 			if err := getJSON(c.URL(i), "/_status/gossip/local", &m); err != nil {
 				return err
 			}
-			infos := m["infos"].(map[string]interface{})
+			infos, ok := m["infos"].(map[string]interface{})
+			if !ok {
+				return errors.New("no infos yet")
+			}
 			if err := f(infos); err != nil {
 				return util.Errorf("node %d: %s", i, err)
 			}

--- a/acceptance/main_test.go
+++ b/acceptance/main_test.go
@@ -161,10 +161,5 @@ func getJSON(url, rel string, v interface{}) error {
 		}
 		return err
 	}
-	if err := json.Unmarshal(b, v); err != nil {
-		if log.V(1) {
-			log.Info(err)
-		}
-	}
-	return nil
+	return json.Unmarshal(b, v)
 }

